Cloudinary/Classes/Core/Features/Uploader/RequestParams/CLDUploadRequestParams.swift

```diff
@@ -981,6 +985,23 @@ import Foundation
         return self
     }
     
+    /**
+     Set a boolean parameter that determines whether to retrieve detected text information in the uploaded image file. default is false.
+     
+     - parameter enable:        The boolean parameter.
+     
+     - returns:                 The same instance of CLDUploadRequestParams.
+     */
+    @discardableResult
+    open func setOcr(_ enable: Bool) -> Self {
```

Comment:
We need another overload that accepts a string to support new ocr types in the future (so we don't have to publish a version just to add another type).

Reply:
@nitzanj not sure we should do the change now, we will need to add content for the parser in any case. Maybe we should delay this change to that time as well?

Reply:
Not necessarily since we usually just add params (e.g. `adv_ocr:123`) and in such cases the result key will still just be `adv_ocr` so it won't require any changes. 

It will also be more consistent with other SDKs where we pass a string so it'll make documentation easier for the doc teams.

Reply:
@nitzanj ok, will add a new function:

`open func setOcrType(_ type: String?) -> Self`

passing nill clears the value

or do we need to keep the same function name

what do you think?

Reply:
It should be the same name (so an overload) for consistency with other SDKs, even it it makes the name a bit less accurate.